Omada Controller Install Error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Omada Controller

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/omada.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Simply,Doesn't Install
[ERROR] in line 39: exit code 0: while executing command curl -fsSL "https://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1w-0+deb11u2_amd64.deb" -o "/tmp/libssl.deb

### 🔄 Steps to reproduce the issue.

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/omada.sh)"



### ❌ Paste the full error output (if available).

  ✔️  Installed Azul Zulu Java
 ⠋curl: (22) The requested URL returned error: 404
 ⠙
[ERROR] in line 39: exit code 0: while executing command curl -fsSL "https://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1w-0+deb11u2_amd64.deb" -o "/tmp/libssl.deb"
curl: (22) The requested URL returned error: 400
/dev/fd/63: line 74: SPINNER_PID: unbound variable

### 🖼️ Additional context (optional).

--- a/install/omada-install.sh
+++ b/install/omada-install.sh
@@ -36,7 +36,7 @@ msg_ok "Installed Azul Zulu Java"
 
 msg_info "Installing libssl (if needed)"
 if ! dpkg -l | grep -q 'libssl1.1'; then
-  curl -fsSL "https://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1w-0+deb11u2_amd64.deb" -o "/tmp/libssl.deb"
+  curl -fsSL "https://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1w-0+deb11u3_amd64.deb" -o "/tmp/libssl.deb"
   $STD dpkg -i /tmp/libssl.deb
   rm -f /tmp/libssl.deb
   msg_ok "Installed libssl1.1"

